dotnet-desktop-guide/net/winforms/controls-design/overview.md

```diff
@@ -38,16 +53,17 @@ Deciding which kind of control to create can be confusing. This article highligh
             <li>You want to combine the functionality of several Windows Forms controls into a single reusable unit.</li>
         </ul>
     </td>
-    <td><a href="#composite-controls">Composite control</a> by inheriting from <a href="/dotnet/api/system.windows.forms.usercontrol">System.Windows.Forms.UserControl</a>.</td>
+    <td>Design a <a href="#user-controls">user control</a> by inheriting from <a href="/dotnet/api/system.windows.forms.usercontrol">System.Windows.Forms.UserControl</a>.</td>
   </tr>
   <tr>
     <td>
         <ul>
             <li>Most of the functionality you need is already identical to an existing Windows Forms control.</li>
+            <li>You want to group one or more controls into a single reusable control.</li>
```

Comment:
If you're extending an existing WinForms control, are you really grouping one or more things into a single component?